app/main/views.py

```diff
@@ -70,6 +72,25 @@ def update_editor_contents():
 
     return 'OK', 200
 
-@main.route('/resource-view')
+@csrf.exempt
+@main.route('/resource-view', methods =['POST'])
+def post_rating():
+    if request is not None:
+            time = datetime.now()
+            star_rating = request.json['rating']
+            comment = request.json['review']
+            if comment and star_rating:
+                rating = Rating(submission_time=time,
+                                rating=star_rating,
+                                review=comment)
+                db.session.add(rating)
+                db.session.commit()
+            elif star_rating:
+                rating = Rating(rating = star_rating)
```

Comment:
we also probably want to set `submission_time` here too
and no spacing between the `=`